admin/spec/back-end/controllers/questions.spec.js

```diff
@@ -132,11 +135,14 @@ describe('Questions controller', () => {
       expect(getConfigSpy).toHaveBeenCalledTimes(1)
       expect(jwtSpy).toHaveBeenCalledTimes(1)
       expect(prepareQuestionDataSpy).toHaveBeenCalledTimes(1)
-      done()
     })
   })
 
   describe('unhappy paths', () => {
+    beforeEach(() => {
+      spyOn(winston, 'error') // ignore winston messages from unhappy outcomes
```

Comment:
This bit sounds risky - if an error is thrown by winston then something unexpected must have happened?

Reply:
this is covered in PBI 23255

Reply:
Not really.  Winston is only od a logger, and this prevents spurious console output during the test run.   Is there a better method?